scriptmodules/libretrocores/lr-vecx.sh

```diff
@@ -20,8 +20,12 @@ function sources_lr-vecx() {
 }
 
 function build_lr-vecx() {
+    local __flags
```

Comment:
Please remove __ prefix. We reserve that for globals.

Reply:
Ok, replaced.